core/src/main/java/org/mapfish/print/output/Values.java

```diff
@@ -91,6 +91,7 @@ public Values() {
         // nothing to do
     }
 
+    // CSOFF: ParameterNumber
```

Comment:
Not a big fan of hiding this issue. Suggestions: Maybe a small refactoring to group related objects or/and a factory method.

Reply:
Then how to fix that without obfuscating the code...